Packs/Shodan/TestPlaybooks/playbook-Test_shodan_v2.yml

```diff
@@ -544,7 +544,7 @@ tasks:
       - "10"
     scriptarguments:
       seconds:
-        simple: "10"
+        simple: "60"
```

Comment:
Increase the sleep to 60 seconds to avoid 429 status code (Too Many Requests error)
https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/14358973#L1019